web/src/pages/admin/ProjectsAdmin.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable */
```

Comment:
Weird that git reports this as a delete and create rather than a move

Reply:
Yeah no idea why